src/app/docs/filter/demo/inline-filter-demo.component.html

```diff
@@ -0,0 +1,63 @@
+<sky-toolbar>
+  <sky-toolbar-section>
+    <sky-toolbar-item>
+      <sky-filter-button
+        [active]="filtersActive"
+        [showButtonText]="true"
+        (filterButtonClick)="filterButtonClicked()"
+      >
+      </sky-filter-button>
+    </sky-toolbar-item>
+  </sky-toolbar-section>
+</sky-toolbar>
+<sky-filter-inline
+  [hidden]="!showInlineFilters"
+>
+  <sky-filter-inline-item>
+    <label
```

Comment:
Should this label have the `sky-control-label` class? It's really tight on the select field, so I wonder if the class would fix the spacing.

Reply:
Yeah it is a little tight. I had to add a margin class to it as well. Good suggestion.

Reply:
Not sure if it's just me, but the label still seems tight to the input:
![image](https://user-images.githubusercontent.com/7850105/95026194-96672780-065d-11eb-956a-cd4e86a16b59.png)
This is a minor cosmetic issue though, so I'll go ahead and approve. If you think we can tweak something to resolve this, I'm all for it. but we can always just circle back to this later.

Reply:
Good catch. That label was missing a class. Fixed!